python/nano/src/bigdl/nano/pytorch/trainer/Trainer.py

```diff
@@ -250,7 +250,8 @@ def quantize(self, pl_model,  # remove the type requirement for type checking
         """
         Calibrate a Pytorch-Lightning model for post-training quantization.
 
-        :param pl_model:        A Pytorch-Lightning model to be quantized.
+        :param pl_model:        A model to be quantized. Model type can be nn.Module,
+             LightningModule, PytorchONNXRuntimeModel, PytorchOpenVINOModel, etc.
```

Comment:
I don't think the user would understand what `PytorchONNXRuntimeModel` or `PytorchOpenVINOModel` is?

Reply:
OK. Then just tell them any instance of `nn.Module`. All model types we have are subclasses of `nn.Module`.